source/guide_searx.rst

```diff
@@ -0,0 +1,242 @@
+.. highlight:: console
+
+.. author:: FM <git.fm@mmw9.de>
+
+.. tag:: web
+.. tag:: python
+
+.. sidebar:: Logo
+
+
+  .. image:: _static/images/searx.png
+      :align: center
+
+#####
+searX
+#####
+
+.. tag_list::
+
+Searx_ (Wikipedia_) is a free and open-source metasearch engine which aggregate results from more than 70 search services (e.g. Google, Bing etc.) and to avoid user tracking and profiling by these ones.
+
+There are public instances_ of searx available, to get a personal practice before you install your own.
+
+License
+=======
+
+Searx is released under the `GNU Affero General Public License`_.
+
+----
+
+.. note:: For this guide you should be familiar with the uberspace basic concepts of
+
+  * supervisord_
+  * domains_
+  * `web backends`_
+
+Installation
+============
+
+Step 1 - Repository Cloning
+---------------------------
+
+We will prepare the destination for the searx repository:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir -p ~/opt/searx/
+
+And clone the repository from GitHub:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://github.com/searx/searx.git ~/opt/searx/
+
+Step 2 - Python Module Installation
+-----------------------------------
+
+----
+
+.. note:: Here we use python 3. This is the reason, that these guide consider this version aspect to the installation tool like ``pip`` and later to ``python`` it self.
+
+Some python modules are necessary and will be installed in your uberspace with ``pip3`` and the option ``--user``:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ pip3 install --user requests pyyaml pygments werkzeug babel flask flask_babel lxml langdetect python-dateutil
+
+Step 3 - Searx Configuration
+----------------------------
+
+We create the configuration file destination directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/etc/searx
+
+And copy the example file with basic default settings to the new directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cp ~/opt/searx/utils/templates/etc/searx/use_default_settings.yml ~/etc/searx/settings.yml
+
+Now it's time to change some entries in the configuration file:
+
+1. Searx requires for the own instance a secret key. This random number will be created with openssl (16 digits) and placed direct in the config file via sed, the stream editor.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ sed -i -e "s/ultrasecretkey/`openssl rand -hex 16`/g" ~/etc/searx/settings.yml
```

Comment:
While `sed` calls make the guide easier to use right now, it's harder to understand what's going on. Especially because `ultrasecertkey` is not a configuration key, but just an example value, which can randomly change in the future. Please adapt this section to generate a token first and insert it into the file manually, like `ADMIN_TOKEN` in the [bitwarden_rs guide](https://lab.uberspace.de/guide_bitwarden.html#install-bitwarden-rs).

Reply:
Yes, you have right. I considered it in the guide update.